oxlate/headers.py

```diff
@@ -0,0 +1,38 @@
+from .utils import string_to_case_number, case_number_to_string
+
+class Headers:
+    def __init__(self):
+        self._data = {}
+
+    def add(self, name, value, adjust_case_to_allow_duplicates=False):
```

Comment:
Add link to description of case problem?
    https://forums.aws.amazon.com/thread.jspa?messageID=701434